### PR TITLE
SWOT

### PR DESCRIPTION
**Pull request by chhormenghong1**

### CODE REVIEW

File: hello-world/index.html [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <title>hello world</title>
   </head>
   <body>
-    <h1>Hello world!</h1>
```
Comment (zigen) [inline hello-world/index.html:7]: don't remove this line

**Review by zigen — APPROVED** [review]

File: input-counter/main.js [hunk 2]
```diff
@@ -1 +1,26 @@
-// write conde in here
\ No newline at end of file
+// write conde in here
+let num = 0
```
Comment (hrl7) [inline input-counter/main.js]: you need `let`? even you will never change this variable?

File: input-counter/main.js [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+const plusButton = document.getElementById('plus')
+const minusButton = document.getElementById('minus')
+const resetButton = document.getElementById('reset')
+const input = document.getElementById('numberInput')
+const number = document.getElementById('number')
+
+function onChange(symbol){
+    let val = parseInt(input.value)
```
Comment (hrl7) [inline input-counter/main.js]: you need let? even you will never change this variable?

File: input-counter/main.js [hunk 4]
```diff
@@ -1 +1,26 @@
...
+    if (symbol === '+'){
+        number.textContent = (num += val).toString()
+    }
+    else if (symbol === '-'){
+        number.textContent = (num-= val).toString()
```
Comment (hrl7) [inline input-counter/main.js]: add a space before `-=`

**Review by hrl7 — CHANGES_REQUESTED** [review]
use clear name and clean up code: space, indents

**Review by hrl7 — APPROVED** [review]

**Pull request closed without merging**